Dockerfile

```diff
@@ -17,16 +17,20 @@ RUN make bins
 
 ##### Temporal server #####
 FROM ${BASE_SERVER_IMAGE} AS temporal-server
-
-RUN addgroup -g 1000 temporal
-RUN adduser -u 1000 -G temporal -D temporal
-
 WORKDIR /etc/temporal
-
 ENV TEMPORAL_HOME /etc/temporal
 ENV SERVICES "history:matching:frontend:worker"
 EXPOSE 6933 6934 6935 6939 7233 7234 7235 7239
 
+# TODO (alex): switch WORKDIR to /home/temporal and remove "mkdir" and "chown" calls.
+RUN addgroup -g 1000 temporal
+RUN adduser -u 1000 -G temporal -D temporal
```

Comment:
User ID 1000 may conflict with the uid of a real user.  Some Docker implementations require a uid:gid to be passed to the container and those are frequently the uid:gid if the actual user running the container.  Suggest using "nobody" as the uid to run temporal under.

Reply:
@jrsmiley: not sure what you mean here, since the base image never adds a user and `1000` is popular as the first UID to be used.

Reply:
In some Docker implementations designed to avoid allowing root, the user must input a uid:gid that the container will run with.  That means that all uid:gids are possible except for "nobody" and "nogroup".

Reply:
I believe the better practice there is user namespace remapping.
Changing internal UIDs/GIDs is a workaround IMO.